openbb_platform/providers/intrinio/openbb_intrinio/models/forward_ebitda_estimates.py

```diff
@@ -0,0 +1,194 @@
+        List[IntrinioForwardEbitdaEstimatesData],
+    ]
+):
+    """Intrinio Forward EBITDA Estimates Fetcher."""
+
+    @staticmethod
+    def transform_query(
+        params: Dict[str, Any]
+    ) -> IntrinioForwardEbitdaEstimatesQueryParams:
+        """Transform the query params."""
+        return IntrinioForwardEbitdaEstimatesQueryParams(**params)
+
+    @staticmethod
+    async def aextract_data(
+        query: IntrinioForwardEbitdaEstimatesQueryParams,
+        credentials: Optional[Dict[str, str]],
+        **kwargs: Any,
+    ) -> List[Dict]:
+        """Return the raw data from the Intrinio endpoint."""
+        api_key = credentials.get("intrinio_api_key") if credentials else ""
+        BASE_URL = (
+            "https://api-v2.intrinio.com/zacks/ebitda_consensus?"
+            + f"page_size=10000&api_key={api_key}"
+        )
+        symbols = query.symbol.split(",") if query.symbol else None
+        query_str = get_querystring(query.model_dump(by_alias=True), ["symbol"])
+        results: List[Dict] = []
+
+        async def get_one(symbol):
+            """Get the data for one symbol."""
+            url = f"{BASE_URL}&identifier={symbol}"
+            url = url + f"&{query_str}" if query_str else url
+            new_data: List[Dict] = []
+            data = await amake_request(
+                url, response_callback=response_callback, **kwargs
+            )
+            if (
+                not data
+                or not isinstance(data, dict)
+                or not data.get("ebitda_consensus")
+            ):
+                warn(f"Symbol Error: No data found for {symbol}")
+            if isinstance(data, dict) and data.get("ebitda_consensus"):
+                new_data = data.get("ebitda_consensus")  # type: ignore
+                if new_data:
+                    results.extend(new_data)
+
+        if symbols:
+            await asyncio.gather(*[get_one(symbol) for symbol in symbols])
+            if not results:
+                raise EmptyDataError(f"No results were found. -> {query.symbol}")
+            return results
+
+        async def fetch_callback(response, session):
+            """Use callback for pagination."""
+            data = await response.json()
+            messages = data.get("messages")
+            if messages:
+                raise RuntimeError(str(messages))
+            estimates = data.get("ebitda_consensus", {})  # type: ignore
+            if estimates and len(estimates) > 0:
+                results.extend(estimates)
+                while data.get("next_page"):  # type: ignore
+                    next_page = data["next_page"]  # type: ignore
+                    next_url = f"{url}&next_page={next_page}"
+                    data = await amake_request(next_url, session=session, **kwargs)
+                    if (
+                        "ebitda_consensus" in data
+                        and len(data.get("ebitda_consensus")) > 0  # type: ignore
```

Comment:
No need to `get` multiple times the same field. There are a couple of occurences in this in code.